src/Model/SitewideContentReview.php

```diff
@@ -29,13 +30,13 @@ public function updateColumns($itemType, &$columns)
         // {@see SiteTreeContentReview::getOwnerNames()}
         $columns['OwnerNames'] = [
             'printonly' => true, // Hide on page report
-            'title' => _t('SitewideContentReport.Reviewer', 'Reviewer'),
+            'title' => _t(SitewideContentReport::class . '.Reviewer', 'Reviewer'),
         ];
 
         // {@see SiteTreeContentView::getReviewDate()}
         $columns['ReviewDate'] = [
             'printonly' => true, // Hide on page report
-            'title' => _t('SitewideContentReport.ReviewDate', 'Review Date'),
+            'title' => _t(SitewideContentReport::class . '.ReviewDate', 'Review Date'),
```

Comment:
At the moment we need to use `SilverStripe\\SiteWideContentReport\\SitewideContentReport.ReviewDate` because the text collector doesn't support `::class` annotations yet (reference: https://github.com/silverstripe/silverstripe-framework/issues/7647)